thamos/lib.py

```diff
@@ -373,14 +374,20 @@ def advise_here(
     github_base_repo_url: typing.Optional[str] = None
 ) -> typing.Optional[tuple]:
     """Run advise in current directory, requires no arguments."""
-    if not os.path.isfile("Pipfile"):
-        raise FileNotFoundError("No Pipfile found in current directory")
+    requirements_format = thoth_config.requirements_format
+    if requirements_format == "pipenv":
+        if not os.path.isfile("Pipfile"):
+            raise FileNotFoundError("No Pipfile found in current directory")
```

Comment:
I wonder if these exceptions are not suitable for thoth-python rather than here.

Reply:
> I wonder if these exceptions are not suitable for thoth-python rather than here.

I thought the same, but isn't that a reason why they shouldn't get there? :)

Reply:
If they are in thoth-python, it's all at one place once. Otherwise, any component using thoth-python will need to do this check again.

Reply:
Perfect, PTAL https://github.com/thoth-station/python/pull/251 @fridex